core/ustring.cpp

```diff
@@ -3049,6 +3049,23 @@ String String::replacen(const String &p_key, const String &p_with) const {
 	return new_string;
 }
 
+String String::repeat(int p_count) const {
+
+	ERR_FAIL_COND_V_MSG(p_count < 0, "", "Parameter count should be a positive number.");
+
+	String new_string;
+	const CharType *src = this->c_str();
+	int len = length();
```

Comment:
Caching `length()` is not really necessary: the compiler would likely do it for you. Does not hurt to keep it though.

Reply:
I can remove it along with any other changes needed.